tests/functional/api/BackbeatServer.js

```diff
@@ -0,0 +1,102 @@
+const assert = require('assert');
+const http = require('http');
+
+const config = require('../../config.json');
+
+const defaultOptions = {
+    host: config.server.host,
+    port: config.server.port,
+    method: 'GET',
+};
+
+function getUrl(options, path) {
+    return `http://${options.host}:${options.port}${path}`;
+}
+
+describe('Backbeat Server', () => {
+    it('should get a response', done => {
+        const url = getUrl(defaultOptions, '/');
+
+        http.get(url, res => {
+            assert(res.statusCode);
```

Comment:
You could check that status code is what you expect (200?)

Reply:
I think this was a silly test because I was just checking to see if any response was received from the server. But, this is sort of checked in following tests. If error in connection refused, then all tests will just fail. So I'll just remove this test